core/iwasm/libraries/debug-engine/debug_engine.h

```diff
@@ -10,6 +10,8 @@
 #include "gdbserver.h"
 #include "thread_manager.h"
 
+#define DEBUG_EXECUTION_MEMORY_SIZE 0x85000
```

Comment:
Could we define it in core/config.h, and use `#ifndef..#define..#endif`? Since in Embedded system, the memory may be limit, we had better let developer be able to adjust it.

Reply:
Yes, I moved it into `config.h`

BTW, this value is required by lldb, so user shouldn't change it, I leave a comment in config.h as well